remote-workstations/new-agent-vm/rhel-standard-agent.json

```diff
@@ -266,30 +267,7 @@
                     "fileUris": [ "[ concat(parameters('userStorageAccountUri'), 'remote-workstation','/', variables('scriptFileName'))]" ]
                 },
                 "protectedSettings": {
-                    "commandToExecute": "[concat('bash ', variables('scriptFileName'), ' \"', parameters('registrationCode'), '\" \"', parameters('agentType'), '\" \"', variables('vmSettings').vmNamePrefix, '\" \"', parameters('domainToJoin'), '\" \"', parameters('domainUsername'), '\" \"', parameters('domainPassword'), '\" \"', parameters('domainGroupToJoin'), '\" \"', parameters('AgentChannel'), '\"')]",
-                    "storageAccountName": "[parameters('userStorageAccountName')]",
-                    "storageAccountKey": "[parameters('userStorageAccountKey')]"
-                }
-            }
-        },
-        {
-            "type": "Microsoft.Compute/virtualMachines/extensions",
```

Comment:
Getting rid of this was supposed to get rid of the hang...?

Reply:
Yes, it looks like it's the older extension hanging the script.